editor/src/clj/editor/pipeline/bob.clj

```diff
@@ -381,12 +382,13 @@
 ;; Build HTML5
 ;; -----------------------------------------------------------------------------
 
-(defn- build-html5-output-path [project]
+(defn- build-html5-output-path
+  ([project]
   (let [ws (project/workspace project)
-        build-path (workspace/build-path ws)]
-    (io/file build-path "__htmlLaunchDir")))
+        build-path (workspace/build-html5-path ws)]
+    (io/file build-path "__htmlLaunchDir"))))
 
-(def build-html5-bob-commands ["distclean" "build" "bundle"])
```

Comment:
Do not use `distclean` so we don't need to rebuild everything from scratch every time.